Unix/doc/setup-ntlm-omi.md

```diff
@@ -0,0 +1,163 @@
+#Enabling SPNEGO/NTLM Authentication on OMI
+
```

Comment:
This file is long enough. Should you do a table of contents at the top, so if someone wants a specific topic, they can jump right there? Most things in ostc-docs, for examples, have a table of contents, unless it's very short.